src/ReportPortal.Shared/Observers/DefaultReportEventsObserver.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ReportPortal.Client.Abstractions.Models;
+using ReportPortal.Shared.Converters;
+using ReportPortal.Shared.Extensibility;
+using ReportPortal.Shared.Extensibility.ReportEvents;
+using ReportPortal.Shared.Extensibility.ReportEvents.EventArgs;
+using ReportPortal.Shared.Reporter;
+using System.Collections.Generic;
+
+namespace ReportPortal.Shared.Observers
```

Comment:
Let's keep this class somewhere in `Embedded` folder.

Reply:
moved.